nucleus/deploy/model_endpoint.py

```diff
@@ -1,12 +1,41 @@
 import concurrent.futures
 from collections import Counter
-from typing import Dict, Optional, Sequence
+from typing import Any, Dict, Optional, Sequence
 
 TASK_PENDING_STATE = "PENDING"
 TASK_SUCCESS_STATE = "SUCCESS"
 TASK_FAILURE_STATE = "FAILURE"
 
 
+class EndpointRequest:
```

Comment:
since we have a function to predict on multiple inputs, I added this so people could pass in a mix of urls/jsons as inputs. Not sure if that'll happen

Reply:
Might be good to add an `EndpointResponse` class for symmetry?